src/h5web/dataset-visualizer/DimensionMapper.tsx

```diff
@@ -83,9 +87,20 @@ function DimensionMapper(props: Props): JSX.Element {
 
   return (
     <div className={styles.mapper}>
+      <div className={styles.axisMapper}>
+        {rawDims.map((d, i) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <span key={`${i}${d}`}>{d}</span>
+        ))}
+      </div>
```

Comment:
This is to display the length of each dimension as I felt it was helping. The aspect needs to be tuned to be aligned with the corresponding button (a grid layout ?).